dev-tools/compose/docker-compose.discovery.prod.yml

```diff
@@ -12,6 +12,8 @@ services:
     deploy:
       mode: replicated
       replicas: '${DISCOVERY_PROVIDER_REPLICAS}'
+    ports:
+      - '6379:6379'
```

Comment:
for test access, doesn't expose anything in prod / stage

Reply:
audius-d / audius-docker-compose use different docker compose files. the prod flag is for the build step if i understand right

Reply:
FYI @alecsavvy this breaks replicas so we can no longer run multiple DNs locally - is there another way?

Reply:
Fixed in https://github.com/AudiusProject/audius-protocol/commit/ea824b71799a247e538e7e0879764a8ec48a55fb